src/test/ajaxWrapper.test.js

```diff
@@ -39,6 +39,20 @@ describe("ajaxWrapper", function () {
       });
     });
 
+    it("defaults to GET when no method is supplied", function (done) {
+      ajaxWrapper({
+        url: config.apiURL
+      })
+      .success(function (response) {
+        expectAsync(function () {
+          expect(response.body.name).to.equal("Marathon");
+        }, done);
+      })
+      .error(function () {
```

Comment:
The idea is that if the response is a non 2xx as expected the .error function shouldn't also be called.